scripts/utils/analyze.py

```diff
@@ -105,10 +105,10 @@ def __init__(self, tools, full=False):
             },
             {
                 "gname": "random",
-                "tools-in-group": ["tpm2_getrandom"],
+                "tools-in-group": ["tpm2_getrandom", "tpm2_stirrandom"],
                 "tools": [],
                 "conflict": None,
-                "ignore": set()
+                "ignore": set(['I', 'infile'])
```

Comment:
You shouldn't this part of the patch right? Or is this a side-effect of https://github.com/tpm2-software/tpm2-tools/pull/1335?

Reply:
No i just missed it when i squashed / rebased my commits... I will fix that